### ISSUE TITLE
Some minor issues with the UI

### ISSUE THREAD

**Issue by charpplie** [ValveSoftware/Dota2-Gameplay#11162]
### Description

I found some shortcomings in the new armory interface. 

- [x] First of all, the Immortal effects selection for Invoker is broken. There is an option to change effects only for spheres. As a result, Immortal effects cannot be changed for the following abilities: Tornado, Alacrity, Sun Strike and Deafening Blast. 
- [x] Second, Invoker is the only hero that has a Taunt slot in the main hero overview menu. Also, this slot does not show whether or not taunt is worn, it always shows up as if he is not there (even if equipped). 
- [x] Third, on the Heroes tab, when you go to Guides, this tab is not highlighted for some reason, but the next tab (Trends) is highlighted. And when you are in the Trends tab, no tab is highlighted. Everything works stable with the first tab (Heroes). 
- [x] Fourthly, in the armory tab in the Interface->Chat Wheel section, the hint about Unavailable filter is not displayed correctly (it should not be displayed here at all).  This tooltip, as I understand, is not bound to the Unavailable filter itself, but just stands at the end of the line. 
- [x] And at the end, when I want to get to the armory through the old icon (which is next to notifications and Dota+), I have the new Armory tab highlighted, but not the text itself.

### Screenshots

First issue: 
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/95695663/03ed04ef-d200-4774-b622-c2526ecd7115)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/95695663/b532f8f0-ba54-4df7-8378-a286680f938f)
Second issue:
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/95695663/3aba26fb-9abc-4ad0-89d2-052c74321bcc)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/95695663/8bc20336-ec4d-4ea9-8e28-dfce82297f70)
Third issue:
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/95695663/d47e9d73-853f-4a93-a872-d9fd89884d4d)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/95695663/ebf16a8c-c083-478a-bf87-4d92082b2e9b)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/95695663/9be3d4db-0560-49c7-8ab8-096ca04cd993)
Fourth issue:
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/95695663/2f28b2f4-b0a3-4d96-9054-086055c9c58e)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/95695663/31667a32-d624-46ce-8e53-e52a0096cd5a)
last issue:
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/95695663/6e95718b-1112-4faf-aff7-84a23c4ee3e4)

**Comment by Migueacalle98** [comment]
Experiencing same issue

**Comment by charpplie** [comment]
I also want to add a some more shortcomings that I discovered later. Firstly, some Arcana sets are not displayed as a whole set but are reflected only as a separate item. I found such a flaw in the following heroes: Spectre, Ogre Magi and Pudge. And I also found another malfunction related to the new arsenal interface. If you select any tab (with the exception of the main Loadout), and then go to the main menu, go back to the hero overview and try to select any other tab except the one that was opened earlier, then they will overlap. 
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/95695663/735ab98b-b60c-440c-8eac-e188b4d1ca07)
